db/migrate/20240702094708_add_london_fees_attribute_to_claims.rb

```diff
@@ -0,0 +1,5 @@
+class AddLondonFeesAttributeToClaims < ActiveRecord::Migration[7.0]
+  def change
+    add_column :claims, :london_fees, :boolean, null: true
```

Comment:
This might be a bit picky.

The convention in Active Record is that references are singular nouns and then a `has_many` relation is the plural. For this reason, `london_fees` suggest to me that there should be a model `LondonFee` with which the claim model has a one-to-many relation. Is it possible to have a different name? `london` might be enough, or the more wordy `firm_is_in_london`.

Reply:
@jrmhaig thanks for this! As discussed I've refactored the attribute name to london_rates_apply and rolled-back/re-applied the migration locally. All is looking good and tests are passing